app/Http/Controllers/Serivice/ServiceRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+namespace App\Http\Controllers\Service;
+
+use App\Service;
+use Illuminate\Database\Eloquent\Model;
+
+class ServiceRepository
```

Comment:
Add a method to delete a model. At the moment, in the controller you are not using your repository class to delete a model. You may need to make a similar change to other repositories as well.